### PR TITLE
Introduce script to combine npm and bower installs

### PR DESCRIPTION
**Pull request by mok4ry**
`npm run fullinstall` to install both `npm` and `bower` dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "gulp build && node ./build/bin/www.js"
+    "start": "gulp build && node ./build/bin/www.js",
```
Comment (kocsenc) [inline package.json:6]: Are we guaranteed that `build` will exist on first run?

Comment (mok4ry) [inline package.json:6]: `gulp build` ensures that `./build` exists. If `gulp build` fails, the script will short-circuit and the second part won't execute.

Comment (kocsenc) [inline package.json:6]: <3 `bash`

**Comment by kocsenc** [comment]
Might be interesting to think about getting rid of `bower`. Both `bower` and `npm` do the same thing. `npm` also has bootstrap and `fa`.

**Pull request merged**